concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1391,6 +1399,43 @@ jobs:
                   cf enable-feature-flag diego_docker
 
       - aggregate:
+        - task: deploy-compose-service-broker
+          config:
+            platform: linux
+            image_resource:
+              type: docker-image
+              source:
+                repository: governmentpaas/cf-cli
```

Comment:
This needs the version pinning to match the current changes in master from #910.